server/src/main/java/server/BazelLanguageServer.java

```diff
@@ -37,22 +38,28 @@ public BazelLanguageServer() {
 
     @Override
     public CompletableFuture<InitializeResult> initialize(InitializeParams params) {
-        // NOTE:
-        // Don't place logging messages inside this function. They will cause the
```

Comment:
LOL! This was staring us in the face the whole time.

Reply:
ARE YOU F'ING KIDDING ME????????

Reply:
HAHAHHAHAHA